cli/pack/common_test.go

```diff
@@ -26,10 +28,10 @@ func TestGetPackageFullname(t *testing.T) {
 	assert.Equal("myapp-1.2.3-4.tar.gz", getPackageFullname(&ctx))
 
 	ctx.Pack.Type = RpmType
-	assert.Equal("myapp-1.2.3-4.rpm", getPackageFullname(&ctx))
+	assert.Equal(fmt.Sprintf("myapp-1.2.3-4.%s.rpm", runtime.GOARCH), getPackageFullname(&ctx))
 
 	ctx.Pack.Type = DebType
-	assert.Equal("myapp-1.2.3-4.deb", getPackageFullname(&ctx))
+	assert.Equal(fmt.Sprintf("myapp_1.2.3-4-1_%s.deb", runtime.GOARCH), getPackageFullname(&ctx))
```

Comment:
What looks strange for me: `4` is the distribution version on RPM based distros, but `1` is the distribution version on the Deb based distros.